tools/docker/run.sh

```diff
@@ -108,7 +103,7 @@ main() {
 
     # Save the container name so that it can easily be stopped/removed later
     echo "$NAME" >> "${scriptdir}/.test_containers"
-    run docker container run "${DOCKEROPTS[@]}" "prplmesh-runner$TAG" "$@"
+    run docker container run "${DOCKEROPTS[@]}" "${image}" "$@"
```

Comment:
Why not build a local image if the pull fails?

Reply:
As explained in the commit message:

    docker: remove auto-building of images
    
    Since images are now pulled automatically, it is not needed to build
    them automatically if they don't exist. Remove that feature.

The automatic building only happens the very first time - at that point, you could just as well pull instead of build, so it's kind of pointless.

It would make sense to build every time. But then I think we should have a script that does the entire build-image, build prplmesh, run tests flow. Building the images just takes 4s on my laptop when everything is cached, so it's OK-ish to do it every time.